pkg/kubectl/cmd/util/helpers.go

```diff
@@ -400,3 +401,21 @@ func UpdateObject(info *resource.Info, updateFn func(runtime.Object) error) (run
 	}
 	return info.Object, nil
 }
+
+type groupID int
+
+const (
+	unknownGroup groupID = iota
+	apiGroup
+	experimentalGroup
+)
+
+func groupForResource(resource string) groupID {
```

Comment:
This isn't really cmd/util/helpers, and is also questionable.  Why not solve this correctly by exposing a new method on RESTMapper for GroupForResource that returns a string "api", "experimentalapi"?